lib/rules/web/admin_console/4.12/monitoring_alerts.xyaml

```diff
@@ -345,7 +345,7 @@ show_alert_graph:
 hide_show_alert_graph_button:
   element:
     selector:
-      xpath: //h2[@class='co-section-heading']/../button
+      xpath: //div[@class="pf-c-toolbar__item"]/button
```

Comment:
Do we have any other property can be used to identify the button instead of `class` ? class name can easily change

Reply:
I update the new xPath as above, but case is back to the same failures as before. It seems that case can not find the button using the updated xPath. I double-checked xPath on admin-console, it still works.

```
10-13 14:35:11.306      Then the step should succeed                                                           # features/step_definitions/common.rb:4
10-13 14:35:11.306        the step failed (RuntimeError)
10-13 14:35:11.306        /home/jenkins/ws/workspace/ocp-common/Runner/features/step_definitions/common.rb:6:in `/^the step should( not)? (succeed|fail)$/'
10-13 14:35:11.306        features/tierN/web/monitoring/alerts-browser.feature:797:in `the step should succeed'
```

@lihongyan1 could you please take a look?
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/644642/console
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/644713/console